src/neuroglancer/chunk_manager/frontend.ts

```diff
@@ -156,6 +158,26 @@ export class ChunkQueueManager extends SharedObject {
         source.deleteChunk(chunkKey);
       }
       visibleChunksChanged = true;
+    } else if (update['requestorChunkKey'] !== undefined) {
+      const rpc = this.rpc!;
+      const source: ChunkSource = rpc.get(update['source']);
+      const requestorChunkKey: string = update['requestorChunkKey'];
+      const requestorRef: any = update['requestorRef'];
+      const key = update['id'];
+
+      const chunk = source.chunks.get(key);
+      const data = chunk ? (<any>chunk)['data'] : undefined;
+      let message: string|undefined = undefined;
+      if (!data) {
+        if (chunk) {
+          message = 'Chunk has no data';
+        } else {
+          message = `No chunk found at ${key} for source ${source.constructor.name}`;
```

Comment:
Can this error ever occur?

Reply:
In theory, yes, both of these could occur, but they would most likely be due to a programming error. The case of an existing chunk with no data would most likely happen if someone were to try to somehow implement a computation on non-volume chunks.